CHANGELOG

```diff
@@ -1,5 +1,10 @@
 # spark-redshift Changelog
 
+## 4.0.0 (2019-07-29)
```

Comment:
Are there any features currently in the library that you don't plan to support in the future?

Reply:
There are features that were in the library that are not supported anymore (but they might work):
- Apparently SaveMode.ErrorIfExists, SaveMode.Ignore and SaveMode.Append tests are failing, so those options don't work. This is probably the biggest loss.
- Postgres driver integration
- IAM integration

Should I mention this in the CHANGELOG ? Should we fix at least SaveModes before going to 4.0.0 ?

Reply:
My advice is to remove these features before going to 4.0.0, with the intention of adding them back (and making them work) if anyone asks for them. It's not your fault that these features are untested and potentially broken, but users of the library won't see it that way.

Reply:
So I'd say, make this another snapshot release, and then the next thing you should do is remove the untested features and release that as 4.0.0. You could note in the changelog that library maintainers would be happy to work on adding these features back if anyone requests them.

Reply:
(Also, I don't know how important SaveModes are. If they're important, we should fix that feature, not remove it.)

Reply:
Allright, I'll publish another `preview` version for now. We can look into SaveMode later if needed